src/fsharp/FSharp.Core/control.fs

```diff
@@ -1536,43 +1536,35 @@ namespace Microsoft.FSharp.Control
     // Contains helpers that will attach continuation to the given task.
     // Should be invoked as a part of protectedPrimitive(withResync) call
     module TaskHelpers = 
-        let private continueWithExtra token =
-#if FSCORE_PORTABLE_OLD || FSCORE_PORTABLE_NEW
-            token |> ignore
-            TaskContinuationOptions.None
-#else
-            token
-#endif
-
         let continueWith (task : Task<'T>, args, useCcontForTaskCancellation) = 
 
             let continuation (completedTask : Task<_>) : unit =
                 args.aux.trampolineHolder.Protect((fun () ->
                     if completedTask.IsCanceled then
                         if useCcontForTaskCancellation
-                        then args.aux.ccont (new OperationCanceledException(args.aux.token))
+                        then args.aux.ccont (new OperationCanceledException())
```

Comment:
Is this an intrinsic part of this fix?  Or is only https://github.com/Microsoft/visualfsharp/pull/3255/files#diff-34ee82347e9e0b47265fcecaa4850174R1567 needed?  If the latter, please remove this change to keep the changes entirely separate,